plugin/evm/syncervm_client.go

```diff
@@ -97,6 +106,12 @@ type StateSyncClient interface {
 	ClearOngoingSummary() error
 	Shutdown() error
 	Error() error
+
+	// Methods to try to enable dynamic state sync
```

Comment:
nit: can avoid the "try" (tentative) language so the draft work is closer to the complete work